docs/src/language_servers.md

```diff
@@ -491,12 +491,16 @@ There are multiple options:
 
 1. Install the sorbet and sorbet-runtime gem (see [github:sorbet/sorbet](https://github.com/sorbet/sorbet)):
 
-        gem install sorbet
-        gem install sorbet-runtime
+    ```sh
```

Comment:
I've oriented myself on the other two LSP-clients, maybe we should add the `sh` to their fences, too?

Reply:
Not sure I get what you mean. I thought I've added it to all Ruby clients.